ts/tests/testSongParser.ts

```diff
@@ -0,0 +1,78 @@
+// We'll probably end up moving some of these functions to the parser since we're storing a new JSON format
+import {reverseString, renderChord, renderChordFingering, renderChordLyricLine, renderLines, instrumentsData} from "../songParser";
+import { assert, expect } from "chai";
+
+describe("reverseString", () => {
+  it("should reverse string", () => {
+    const string = "2000";
+    assert(reverseString(string) === "0002");
```

Comment:
`assert.equal` or `assert.strictEqual` is better because it shows you what the two sides are if they don’t match.